src/constants/constants.js

```diff
@@ -5,9 +5,11 @@ const { KEG_GLOBAL_CONFIG } = process.env
 
 // Cache the root of the CLI for easy access
 const CLI_ROOT = path.join(__dirname, '../../')
+const CONFIG_FOLDER_NAME = '.kegConfig'
```

Comment:
Moving the folder name to its own variable is fine, but I don't see the of also adding `CLI_ROOT_CONFIG_FOLDER`
  * There's no longer a single location where the `gobalConfig` is loaded from, and makes other methods that need it more complicated. 
  * I may be missing something, but previously it was setup so that
    * If `KEG_GLOBAL_CONFIG` env exist, the `globalConfig` would be loaded from it
    * Otherwise if would use the default `~/.kegConfig` folder
    *  Based on this, I don't see the need for `CLI_ROOT_CONFIG_FOLDER`

Reply:
removed this; not necessary